packages/sdk/test/unit/sdkBase.spec.ts

```diff
@@ -584,6 +584,32 @@ describe("NxtpSdkBase", () => {
       expect(res.bid).to.be.eq(auctionBid);
       expect(res.bidSignature).to.be.eq(bidSignature);
     });
+
+    it("happy: should sort multiple transfer quotes", async () => {
+      const { crossChainParams, auctionBid, bidSignature } = getMock();
+
+      recoverAuctionBidMock.returns(auctionBid.router);
+      transactionManager.getRouterLiquidity.resolves(BigNumber.from(auctionBid.amountReceived));
+
+      setTimeout(() => {
+        messageEvt.post({
+          inbox: "inbox",
+          data: { bidSignature, bid: { ...auctionBid, amountReceived: "100000" }, gasFeeInReceivingToken: "0" },
+        });
+        messageEvt.post({
+          inbox: "inbox",
+          data: { bidSignature, bid: { ...auctionBid, amountReceived: "100002" }, gasFeeInReceivingToken: "0" },
+        });
+        messageEvt.post({
+          inbox: "inbox",
+          data: { bidSignature, bid: { ...auctionBid, amountReceived: "100004" }, gasFeeInReceivingToken: "0" },
```

Comment:
do we need to test gasFeeInReceivingToken being different?

Reply:
p sure no but just double checking

Reply:
No that's just a convenience thing it's already subtracted.

Reply:
got it